### DISCUSSION TITLE
Repeats for full map

### DISCUSSION THREAD

**Discussion by heisenberg2980** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#260] [Q&A] [answered]
Is it possible to clean the whole map multiple times? I have seen this repeat options for rooms or zones but I would like the option to do the same with the whole map (i.e. when I want the robot to clean all the house multiple times)

**Comment by PiotrMachowski** [comment, marked as answer]
I haven't seen this option both in HA or Xiaomi app, so I don't think it's possible. You can create one big zone instead though

**Reply by heisenberg2980** [reply]
Yeah, you are right, I just thought it could be a good addition to this amazing card, and even exceed the functionality the original Xiaomi app provides. For now I will just add a new input_number to my card and will use it in an automation to start cleaning again every time the robot completes the clean until it reaches the number
